code-of-conduct.md

```diff
@@ -16,7 +16,7 @@ We expect all members of the Camunda community, including administrators, users,
 
 The Camunda Code of Conduct applies to all our Camunda projects and products, including but not limited to Camunda Platform 7, Camunda Cloud, the Camunda Community Hub, and bpmn-io. It also applies to all Camunda events. This includes official Camunda events, such as CamundaCon, Camunda Process Automation Forums, Camunda meetups, trainings, workshops, and unofficial Camunda events.
 
-The Camunda Community Code of Conduct also applies to any Camunda online space. This includes Slack workspaces, discussion forums, websites, code repositories, mailing lists, and more. The Camunda Community Code of Conduct also applies to private correspondence between community members through our online spaces.
+The Camunda Community Code of Conduct also applies to any Camunda online space. This includes discussion forums, websites, code repositories, mailing lists, and more. The Camunda Community Code of Conduct also applies to private correspondence between community members through our online spaces.
```

Comment:
Just noticed this was an update to the Code of Conduct. We probably want to keep this in because we still have the Champion/meetup community Slack.